tests/sod723.test.ts

```diff
@@ -0,0 +1,9 @@
+import { test, expect } from "bun:test"
+import { convertCircuitJsonToPcbSvg } from "circuit-to-svg"
+import { fp } from "../src/footprinter"
+
+test("sod723", () => {
+  const soup = fp.string("sod723").circuitJson()
```

Comment:
```suggestion
  const circuitJson = fp.string("sod723").circuitJson()
```

Don't use soup as variable anymore